src/modules/messages/index.ts

```diff
@@ -34,6 +37,24 @@ export class Messages {
     return (await response).data.createMessage as MessagesInterface;
   }
 
+  public async updateMessage(
+    id: string,
+    input: MessageUpdateInputInterface
+  ): Promise<MessagesInterface> {
+    const response = this.client.mutate({
+      mutation: UPDATE_MESSAGE_MUTATION,
+      variables: { input: input, id: id},
+    });
+    return (await response).data.updateMessage as MessagesInterface;
+  }
+
+  public async deleteMessage(id: string): Promise<void> {
+    await this.client.mutate({
+      mutation: DELETE_MESSAGE_MUTATION,
+      variables: { id: id },
```

Comment:
change: we should return at least something like true or false if is the message was deleted or not.